go/pkg/sdk/optimism/sdk_optimism.go

```diff
@@ -268,6 +268,72 @@ func (sdk *SdkOptimism) GetSlippage(stable constants.Stablecoin, amount *big.Int
 	return percentageChange, expectedPriceFloat, executedPriceFloat, nil
 }
 
+func(sdk *SdkOptimism) createSwapData(callTo common.Address, approveTo common.Address, sendingAssetId common.Address, receivingAssetId common.Address, fromAmount *big.Int, fromStableTosDai bool) ([]optimismContracts.LibSwapSwapData, error) {
```

Comment:
the code of this function is not well indented. we have pre-commit hooks to check that, but the readme doesn't mention them. here's what you need to do - at the root level - to install them:

```bash
pre-commit install
```

Reply:
then you can apply pre-commit hooks on all files, it will automatically modify them to create a code following the standard of [golangci-lint](https://golangci-lint.run/).